scripts/python/bccAccepted.py

```diff
@@ -1,11 +1,13 @@
 import os
+import datetime
+import pathlib
 
 totalSites = 0
 totalBCC = 0
 
 index = 1
 
-dirPath = "../_data"
+dirPath = "../../_data"
```

Comment:
os.path.join("..", "..", "data")

Usually, os.pathsep is used rather than a literal /